src/typings/sharify.d.ts

```diff
@@ -39,6 +39,8 @@ declare module "sharify" {
       IS_MOBILE: boolean
       readonly IP_ADDRESS: string
       readonly METAPHYSICS_ENDPOINT: string
+      readonly NETWORK_CACHE_SIZE: string
+      readonly NETWORK_CACHE_TTL: string
```

Comment:
Aren't these integers?

Reply:
the issue is that all items that come through `process.env` are strings. JS is often smart enough to cast, but its probably best to be honest and to manually cast when needed.